### DISCUSSION TITLE
Meeting Notes

### DISCUSSION THREAD

**Discussion by VanessaMedinaUNR** [Techwise-Team-15/my_pet#35] [Announcements]
The discussion will store recaps from meetings we have through out the following weeks! 

Feel free to leave questions on clarification! However, our project board will have better in-depth task descriptions than these summaries. These mostly will serve for tracking our progress and reminding everyone of changes, updates and ect!

**Comment by VanessaMedinaUNR** [comment]
meeting of July 9th, 2023 (2hrs):

-What has been accomplished so far or almost done: the main menu page, the start page, color scheme decided, video game theme decided, game logo decided, backgrounds created, and organizational systems created 
-What still needs development: fully capability of sprite animations, config page, choosing the pet screen, using the mouse, and further gameplay
-What has been postponed and labeled enhancement: the loading page (saving gameplay and tracking offline time), gameplay of the raccoon and mudskipper, autosaving, music, and sound effects

-Discussed minimal product requirements:
	*due date for minimal product is July 31st 
		
	*enhancements will begin after minimal product completion

	*start and quit gameplay with quiting results in death 
		
	*only pet rock gameplay 

	*activities only on one screen (home):

	*activities: bath, feeding, drinking water and playing

-What is due by wednesday: google slides for presentation, main menu with cosmetic changes, choosing a pet screen fully done for the pet rock, home design complete, completing the config page with theme items and sprite animations
-Decided screen size will be 1920 pixels x 1080 pixels
-Reminders:
 	*new meeting created for Friday, July 14 at 6pm est
	*create issues on github instead of just creating tasks on project board
	*always keep updating the project board with your commit hashes and comments
	*make your own branch for your ticket so you have space to work on your tasks without interrupting main code and other's work
	*when working, it doesn't hurt to let others know so they can possibly join you!

**Comment by dinaolmelak** [comment]
Thank you @VanessaMedinaUNR for the notes. However, the date for our `first release` is July 31, because that way after July 31 we can use the remaining time until Aug 31 for enhancements.

**Reply by VanessaMedinaUNR** [reply]
I accidentally wrote August, so sorry! It's been fixed! @dinaolmelak

**Reply by dinaolmelak** [reply]
No worries, Thank you!
